backend/src/services/rubricService.ts

```diff
@@ -0,0 +1,57 @@
+import { Rubric } from "../../../palette-types/src/DatabaseSafeTypes";
```

Comment:
since `rubricService` is an interface, should we move it to the palette-types module? That way it's also close to the DatabaseSafeTypes declaration